benchmark/_http-benchmarkers.js

```diff
@@ -173,7 +173,7 @@ class H2LoadBenchmarker {
   }
 
   processResults(output) {
-    const rex = /(\d+(?:\.\d+)) req\/s/;
+    const rex = /(\d+\.\d+) req\/s/;
```

Comment:
Maybe the original intent was to handle integer values? I don't know if that's necessary, please disregard if we know that doesn't happen.
```suggestion
    const rex = /(\d+(?:\.\d+)?) req\/s/;
```

Reply:
I think it shouldn't work with integers. The h2load output is something like finished in 233.18s, **0.43 req/s**, 105B/s